splunk-otel-android/src/main/java/com/splunk/rum/Config.java

```diff
@@ -266,6 +298,20 @@ public Builder anrDetectionEnabled(boolean enable) {
             return this;
         }
 
+        /**
+         * Configures the rate at which frame render durations are polled.
+         * @param interval - The period that should be used for polling
+         * @return this
+         */
+        public Builder slowRenderPollingDuration(Duration interval){
```

Comment:
WDYT about using the `SlowRenderingDetector` name here as well? (and just using the same name for this feature everywhere in code). E.g. `slowRenderingDetectorPollingInterval()`, `slowRenderingDetectorEnabled()`